content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -42,21 +42,22 @@ refs/tags/baseline-experiment:
 ```
 
 This command lists remote experiments based on that repo's `HEAD`. You can use
-`--all` to list all experiments, or add any other supported option to the remote
-`dvc exp list` command.
+`--all-commits`/`-A` to list all experiments, or add any other supported option
+to the remote `dvc exp list` command.
 
 [shared]: /doc/user-guide/experiment-management/sharing-experiments
 
 ## List experiment names to use in scripts
 
 `dvc exp list` may be printing too much information when it comes to feed its
 output to other commands. You can get only the names of the experiments via the
-`--name-only` flag. For example, to get all the experiment names from a remote
-(`origin`):
+`--name-only` flag. For example, to download the model files from all
+experiments in a remote:
 
 ```dvc
-$ for experiment in $(dvc exp list origin --name-only --all) ; do
-  dvc exp pull "${experiment}"
+$ for name in $(dvc exp list origin --name-only -A); do
+    dvc get --rev ${name} git@github.com:iterative/example-dvc-experiments.git \
```

Comment:
Opened https://github.com/iterative/dvc/issues/7814 to request shortening the Git URL here.